packages/zowe-explorer/__tests__/__integration__/DatasetTree.integration.test.ts

```diff
@@ -21,6 +21,7 @@ import * as sinon from "sinon";
 import * as chai from "chai";
 import * as chaiAsPromised from "chai-as-promised";
 import * as globals from "../../src/globals";
+const Buffer = require("safer-buffer").Buffer;
```

Comment:
We normally don't use require statements like this in the code base. Is there a better way? Or perhaps we can remove the fix for deprecated buffer in a different PR since it affects not just this module.

Reply:
Using it with require() seems to be the intended way.
Removed it for now.